### ISSUE TITLE
FTBFS on jessie: texinfo error

### ISSUE THREAD

**Issue by mark0n** [epicsdeb/rtems-binutils#1]
```
./chew -f ../../../bfd/doc/doc.str <../../../bfd/doc/../mmo.c >mmo.tmp
/bin/bash ../../../bfd/doc/../../move-if-change mmo.tmp mmo.texi
restore=: && backupdir=".am$$" && \
rm -rf $backupdir && mkdir $backupdir && \
if (makeinfo --split-size=5000000 --split-size=5000000 --version) >/dev/null 2>&1; then \
  for f in bfd.info bfd.info-[0-9] bfd.info-[0-9][0-9] bfd.i[0-9] bfd.i[0-9][0-9]; do \
    if test -f $f; then mv $f $backupdir; restore=mv; else :; fi; \
  done; \
else :; fi && \
if makeinfo --split-size=5000000 --split-size=5000000   -I ../../../bfd/doc \
 -o bfd.info `test -f 'bfd.texinfo' || echo '../../../bfd/doc/'`bfd.texinfo; \
then \
  rc=0; \
else \
  rc=$?; \
  $restore $backupdir/* `echo "./bfd.info" | sed 's|[^/]*$||'`; \
fi; \
rm -rf $backupdir; exit $rc
../../../bfd/doc/bfd.texinfo:326: unknown command `colophon'
../../../bfd/doc/bfd.texinfo:337: unknown command `cygnus'
../../../bfd/doc/elf.texi:11: raising the section level of @subsubsection which is too low
Makefile:394: recipe for target 'bfd.info' failed
make[4]: *** [bfd.info] Error 1
make[4]: Leaving directory '/home/vagrant/rtems-binutils/build-arm/bfd/doc'
```

It seems like this bug has been fixed upstream with commit 7ca01ed925b6be550d5d46bd2f0a68bb3e63ff7b which has been released with binutils-2.24. See http://www.mail-archive.com/bug-binutils@gnu.org/msg14429.html for details. Does it make sense to upgrade to the latest binutils code?

**Comment by mark0n** [comment]
According to the [RTEMS website](https://devel.rtems.org/wiki/Release/4.9) RTEMS 4.9 (which is still the default since according to Michael 4.10 does not work with everything, yet) is supposed to be build with binutils 2.19.

**Comment by mark0n** [comment]
Fixed by 8f6cd7246b8f626b00fc46abca3275664dde4533. Michael, thanks for providing this workaround!
